### PR TITLE
Discuss pip install --user option; TensorFlow and PyTorch stylization 

### PR DESCRIPTION
**Pull request by felker**

### CODE REVIEW

File: docs/theta-gpu/data-science-workflows/building-python-packages.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ To build Python packages for ThetaGPU, there are two options: build on top of a
 ## Build on ThetaGPU compute using Conda
 To build on ThetaGPU compute and install your own packages, login to theta and then submit an interactive job to log on to ThetaGPU compute node. 
 
-Please see [Running Pytorch with Conda](https://www.alcf.anl.gov/support/user-guides/theta-gpu/data-science/dl-frameworks/running-pytorch-conda/index.html) or [Running Tensorflow with Conda](https://www.alcf.anl.gov/support/user-guides/theta-gpu/data-science/dl-frameworks/running-tensorflow-conda/index.html) for more information.
+Please see [Running PyTorch with Conda](https://www.alcf.anl.gov/support/user-guides/theta-gpu/data-science/dl-frameworks/running-pytorch-conda/index.html) or [Running TensorFlow with Conda](https://www.alcf.anl.gov/support/user-guides/theta-gpu/data-science/dl-frameworks/running-tensorflow-conda/index.html) for more information.
```
Comment (saforem2) [inline docs/theta-gpu/data-science-workflows/building-python-packages.md:7]: Should we use links to the public page or use relative links in the repo?

I don't have much of an opinion either way, and have seen arguments for both

Comment (felker) [reply docs/theta-gpu/data-science-workflows/building-python-packages.md:7]: no clue, I didnt change them from whoever wrote it originally

Comment (saforem2) [reply docs/theta-gpu/data-science-workflows/building-python-packages.md:7]: fair enough, might as well leave as is then 🤷‍♂️

**Review by saforem2 — APPROVED** [review]
LGTM 👍

**Pull request merged**